doc/glossary.rst

```diff
@@ -0,0 +1,85 @@
+
+    Trough (``time_trough``)
+        The time between the previous decay and the current rise.
+
+    Rise (``time_rise``)
+        The time between the current peak and the next trough.
+
+    Decay (``time_decay``)
+        The time between the current peak and the last trough.
+
+    Rise-decay symmetry (``time_rdsym``)
+        The fraction of the period in the rise phase.
+
+    Peak-trough symmetry (``time_ptsym``)
+        The fraction of the period in the peak phase.
+
+    Sinusoidality
+        A symmetrical wave with 0.5 rise-decay and peak-trough symmetry.
+
+    Voltage Peak (``volt_peak``)
+        The voltage at the current peak.
+
+    Voltage Trough (``volt_trough``)
+        The voltage at the pervious trough.
+
+    Voltage Rise (``volt_rise``)
+        The voltage change between the previous trough and the current peak.
+
+    Voltage Decay (``volt_decay``)
+        The voltage change between the current peak and the next trough.
+
+    Voltage Amplitude (``volt_amp``)
+        The average of the rise and decay voltage.
+
+    Band Amplitude (``band_amp``)
+        The average analytic amplitude of the period or oscillation.
+
+Burst
+-----
+Additional shape features that aid in determining where a signal may be bursting.
+
+.. glossary::
+
+    Amplitude Fraction (``amp_frac``)
+        The average amplitude, relative to all other cycles. A value of 1.0 represents the cycle with
+        the maximum average amplitude. While values approaching 0.0 represents the minimum.
+
+    Amplitude Consistency (``amp_consistency``)
+        The amplitude consistency of a cycle is equal to the maximum relative difference between rises and
+        decay amplitudes across all pairs of adjacent rises and decays that include one of the flanks in the
+        cycle (3 pairs) (e.g. if a rise is 10mV and a decay is 7mV, then its amplitude consistency is 0.7).
+
+    Period Consistency (``period_consistency``)
+        Period consistency is equal to the maximum relative difference between all pairs of adjacent periods
+        that include the cycle of interest (2 pairs: current + previous cycles and current + next cycles) (e.g. if the
+        previous, current, and next cycles have periods 60ms, 100ms, and 120ms, respectively, then the period
+        consistency is min(60/100, 100/120) = 0.6)).
+
+    Monotonicity (``monotonicity``)
+        The monotonicity is the fraction of samples that the instantaneous derivative (numpy.diff) is consistent with
+        the direction of the flank. (e.g. if in the rise, the instantaneous derivative is 90% positive, and in the decay,
+        the instantaneous derivative is 80% negative, then the monotonicity of the cycle would be 0.85 ((0.9+0.8)/2)).
+        The rise and decay flanks of the cycle should be mostly monotonic.
+
+    Burst Fraction (``burst_fraction``)
+        The proportion of a cycle's samples that are bursting according to the dual amplitude threshold algorithm
```

Comment:
is this always / only according to the dual-amp algorithm?
Can it not also reflect bursts detected from the bycycle-burst detection algorithm?

Reply:
Yes, burst fraction is only defined using the dual-amp algorithm, and not the bycycle alogorithm. The bycycle burst detection algorithm returns one value (i.e. amp_fraction, amp_consistency, period_consistency, monotonicity) per cycle. The dual-amp algorithm instead returns one value per sample in a cycle (i.e if a cycle has three samples / timepoints: `np.array([True, True, False])`, the burst fraction would be 0.66.

Reply:
Ahh, I see - the difference is in terms of being defined in terms of samples vs. cycles, which I hadn't tracked properly.